microsoft/testsuites/network/networksettings.py

```diff
@@ -131,7 +132,10 @@ def validate_device_channels_change(self, node: Node, log: Logger) -> None:
                     f"Max channels for device {interface} is <= 1."
                     " Not attempting to change, Skipping."
                 )
+                skip_test = True
```

Comment:
In this kind of pattern, it should be one way setting. It means, if it turns to True, no code should turn it back. If turn it back, it can only check last value, not all.

Reply:
In another word, "all <=1" means "any >1". So, if any > 1, the case won't be skipped.

Reply:
yeah so my thought was that we don't want to mark the full test case as "Skipped" if we tested any of the devices positively? Let me know if that should not be the case

Reply:
The code logic is like below:
```python
skip_test = True

for item in items:
    if item is testable:
        skip_test = False

if skip_test:
   raise SkippedException(....
```